frontend/packages/dev-console/src/components/helm/HelmReleaseResourceTableRow.tsx

```diff
@@ -19,6 +20,20 @@ const HelmReleaseResourceTableRow: React.FC<HelmResourceTableRowProps> = ({
   key,
   style,
 }) => {
+  const status = resource.status?.replicas ? (
+    <Link
+      to={`${resourcePath(
+        resource.kind,
+        resource.metadata.name,
+        resource.metadata.namespace,
+      )}/pods`}
+      title="pods"
+    >
+      {resource.status.replicas || 0} of {resource.spec.replicas} pods
+    </Link>
+  ) : (
+    <Status status={_.get(resource.status, 'phase', 'Created')} />
```

Comment:
do we need a fallback/default here?

Reply:
@abhinandan13jan  what I am trying to do here is, show the no of pods deployed for resources that support it and for others show Created/Running, etc etc. Please have a look at the1st  screenshot in the PR description

Reply:
What I meant is, could there be a case where " of 1 pods" shows up...where the first part could go blank? If there is no chance for that, it's cool...

Reply:
I was trying to point to a case where {resource.spec.replicas} is unavailable and test would be something like "0 of".
However I see that is quite unlikely.